tests/ui/single_element_loop.rs

```diff
@@ -0,0 +1,9 @@
+// Tests from for_loop.rs that don't have suggestions
+
+#[warn(clippy::single_element_loop)]
```

Comment:
We should definitely add a suggestion here. Let me know if you need help with that. You can take a look at `span_lint_and_sugg` usages to see how to do this.

Reply:
Thanks for the suggestion, now it's using `span_lint_and_sugg`